pkg/config/config.go

```diff
@@ -80,6 +81,7 @@ var (
 // Config defines everything needed for apiserver to deal with external services
 type Config struct {
 	JenkinsOptions        *jenkins.Options                   `json:"devops,omitempty" yaml:"devops,omitempty" mapstructure:"devops"`
+	JenkinsClientOptions  *jclient.Options                   `json:"devops,omitempty" yaml:"devops,omitempty" mapstructure:"devops"`
```

Comment:
IMO, `JenkinsClientOptions` is duplicated with `JenkinsOptions`. Please see the CI error.

Reply:
Please use `JenkinsOptions` instead.

Reply:
I have solved this error, and the CI seems all good (except for the codecov error).